commerce-order-web/src/main/resources/META-INF/resources/edit_order.jsp

```diff
@@ -22,7 +22,13 @@ CommerceOrderEditDisplayContext commerceOrderEditDisplayContext = (CommerceOrder
 CommerceOrder commerceOrder = commerceOrderEditDisplayContext.getCommerceOrder();
 
 portletDisplay.setShowBackIcon(true);
-portletDisplay.setURLBack(String.valueOf(renderResponse.createRenderURL()));
+
+if (Validator.isNull(redirect)) {
+	portletDisplay.setURLBack(String.valueOf(renderResponse.createRenderURL()));
+}
+else {
+	portletDisplay.setURLBack(String.valueOf(redirect));
```

Comment:
@Commando18, isn't `redirect` already a `String`? We shouldn't have to call `String.valueOf` on it, right?

Reply:
@Commando18 please fix as requested